addons/hr_holidays/static/src/leave_stats/leave_stats.xml

```diff
@@ -20,14 +20,14 @@
         </div>
         <div t-if="state.department" id="o_leave_stats_department">
             <div class="o_horizontal_separator o_hr_leave_subtitle">
-                <t t-esc="state.department[1]"/>
+                <t t-esc="state.department.display_name"/>
             </div>
             <div t-if="state.departmentLeaves.length === 0">
                 None
             </div>
             <div t-foreach="state.departmentLeaves" t-as="leave" t-key="leave_index" t-attf-class="d-flex flex-row justify-content-between {{leave.sameEmployee ? 'fw-bold': ''}}">
                 <span>
-                    <t t-esc="leave.employee_id[1]"/>: 
+                    <t t-esc="leave.employee_id[1]"/>:
```

Comment:
display_name?

Reply:
The value came from a search_read, I've changed this search_read to web_search_read.